custom-recipes/nlp-analysis-ontology-tagging/recipe.py

```diff
@@ -1,36 +1,32 @@
 # -*- coding: utf-8 -*-
 import dataiku
-from dataiku.customrecipe import get_input_names_for_role, get_output_names_for_role
+from dataiku.customrecipe import get_input_names_for_role
```

Comment:
Perhaps put that too in DkuConfigLoadingOntologyTagging? Since we did it for get_output_names_for_role

Reply:
9622561